packages/checkout/sdk/src/smartCheckout/sell/sell.ts

```diff
@@ -166,9 +168,9 @@ export const sell = async (
   }
 
   const itemRequirements = [
```

Comment:
Thanks for the fix. May I please get improve legibility of this code block. A ternary combined within an array is less clear. we can improve by defaulting a simple `if/else` statement.

```
  const itemRequirements = [];
  if (sellToken?.type === ItemType.ERC1155) {
      itemRequirements.push(getERC1155Requirement(sellToken.id, sellToken.collectionAddress, spenderAddress, sellToken.amount));
  } else {
      getERC721Requirement(sellToken.id, sellToken.collectionAddress, spenderAddress)
  }
```

Reply:
done!